lib/fhir_models/definitions/structures/profiles-types.json

```diff
@@ -2844,7 +2844,7 @@
                 "ele-1"
               ],
               "constraint": [
-               {
+                {
```

Comment:
rubocop excludes the generated files.  It also doesn't do anything to json/xml files since they aren't ruby.